### ISSUE TITLE
Question about training caltech pedestrian dataset 

### ISSUE THREAD

**Issue by AndrewZhao** [weiliu89/caffe#445]
Hi, Liu Wei,
	I have some questions about training SSD using caltech pedestrian dataset. The `num_classes` is 2(person and background).
	Firstly, I convert this dataset to pascal VOC format and the `test.txt` and `trainval.txt` is following:
`test:
000001
000005
000006
000009
000010
000012
000013` 
`trainval:
000002
000003
000004
000007
000008
000011`
	Then, I use `create_list.sh` and `create_data.sh` to generate the text file and lmdb file successfully. And I changed the  label map to 2 classes(person and background).
	At last, I change some parameters(like job file,train data,test data and so on) in `ssd_pascal.py` and conducting this python file.
  	The procedure is OK, **but the Loss is always 0**.
`Iteration 10, loss = 0`
	Could you give me some suggestions about this question.
	Thanks for your kind help.

**Comment by weiliu89** [comment]
https://github.com/weiliu89/caffe/wiki/Train-SSD-on-custom-dataset

**Comment by AndrewZhao** [comment]
@weiliu89 Thanks for prompt reply. I have tried the `ssd.ipynb`. In the `get_labelname`, the assertion goes into effect.My labelmap is:

item {
  name: "none_of_the_above"
  label: 0
  display_name: "background"
}
item {
  name: "person"
  label: 1
  display_name: "person"
}

**Comment by AndrewZhao** [comment]
I solved that problem.Because the bbox coordinates in caltech ped dataset are float numbers. The bbox coordinates must be int in VOC format.

**Comment by MarcosPieras** [comment]
Hi @AndrewZhao, I am working with the SSD to detect people in urban scene. Did you notice any improvement training the SSD with the Caltech pedestrian dataset compared with the SSD model trained by the authors ( with COCO and Pascal ) ?
Thanks!

**Comment by amoussawi** [comment]
Hello @MarcosPieras and @AndrewZhao . You may find some help [here](https://github.com/amoussawi/caffe)

**Comment by XhqGlorry11** [comment]
@amoussawi Hi, I have seen your excellent work. And I 'd like to know where I can get the full text of the essay:An FPGA-Accelerated Design for Deep Learning Pedestrian Detection in Self-Driving Vehicles. Cant find it online for now. Please help! thx~
